aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,285 @@
-.. include:: /../common/stub-topic.txt
+ASP.NET Core Internationalization: Localization and Globalization 
+==================================================================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_ with assistance from `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website with ASP.NET Core will allow your site to reach a wider audience. ASP.NET has :doc:`/fundamentals/request-features` that provide support for localizing into different languages and cultures.
 
-.. _issue: https://github.com/aspnet/Docs/issues/653
\ No newline at end of file
+This sample is based on the **Localization.StarterWeb** project on `GitHub <https://github.com/aspnet/entropy>`__.
+
+.. contents:: In this article:
+  :local:
+  :depth: 1
+  
+.. note:: Currently, resource files are not read when the project is run from Visual Studio. See `this issue <https://github.com/aspnet/dnx/issues/3047>`_ for more information. Until the issue with Visual Studio is addressed, you can test the project by running it from the command line.
+
+IStringLocalizer
```

Comment:
Instead of making this section about a specific interface (which is really what API ref docs are for) I think it would be better make this section about solving a specific problem. Something like "Preparing your application for localization". You could then introduce the `IStringLocalizer` service as a way to setup your application for localization.